.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @renardete @holgiosalos @Valeryibarra
```

Comment:
When Github shows this icon. 
![image](https://user-images.githubusercontent.com/10209180/112413079-9501e800-8ced-11eb-887f-30d5512a4c1f.png)

It tells you that the file does not have the End Of File (EOF). This issue is solved by adding an empty line to the end of the file